lib/btrfs_test.pm

```diff
@@ -18,13 +19,53 @@ sub set_unpartitioned_disk_in_bash {
         $vd = 'sd';
     }
     assert_script_run 'parted --script --machine -l';
-    assert_script_run 'disk=${disk:-$(parted --script --machine -l |& sed -n \'s@^\(/dev/' . $vd . '[ab]\):.*unknown.*$@\1@p\')}';
+    assert_script_run 'disk=${disk:-$(parted --script --machine -l |& sed -n \'s@^\(/dev/' . $vd . 'b\):.*$@\1@p\')}';
```

Comment:
Of course not. Nor did know the reviewer. You should have rather done, what you ask of everyone else: comment on weirdness in the code.